static/js/zamboni/buttons.js

```diff
@@ -274,6 +274,7 @@ var installButton = function() {
             warn(gettext('Not available for your platform'));
             $button.addClass('concealed');
             $button.first().css('display', 'inherit');
+            $button.closest('.item.addon').addClass('incompatible');
```

Comment:
It's always been done with JS, the add-on is output with just the classes `item` and `addon` before this.